script/benchmark

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+set -e
+
+function benchmark() {(
+  cd tmp/bench-site
+  bundle update
+  time bundle exec jekyll build
+)}
+
+rm -rf tmp/benchmark tmp/bench-site
+mkdir -p tmp/bench-site
+
+git clone --depth 1 https://github.com/jekyll/benchmarking.git tmp/benchmark
+
+tmp/benchmark/generate_site.sh 1000 tmp/bench-site
+
+cat > tmp/bench-site/Gemfile << EOF
+gem "jekyll", :path => "."
+gem "minima"
+EOF
+
+benchmark
+
+cat > tmp/bench-site/Gemfile << EOF
+gem "jekyll", :github => "jekyll/jekyll", branch: "master"
```

Comment:
Shouldn't we add:

```
# frozen_string_literal: true 
 
source "https://rubygems.org" 
```

Reply:
OH!!!! 💡

Reply:
`frozen_string_literal` is not a concern here, but `source` is the reason everything kept failing on Travis but not locally 😝

Reply:
...at least, that's what I thought.
https://travis-ci.org/jekyll/jekyll/jobs/338255064#L726

Reply:
> Gems in the groups benchmark, site and development were not installed.